frontend/components/Editor.js

```diff
@@ -1344,7 +1359,7 @@ patch: ${JSON.stringify(
                                 on_selection=${(selected_cell_ids) => {
                                     // @ts-ignore
                                     if (
-                                        selected_cell_ids.length !== this.state.selected_cells ||
+                                        selected_cell_ids.length !== this.state.selected_cells.length ||
```

Comment:
This small bug was devastating! length is always different from array, which means that `onmousemove` changed the `selected_cells` array which means that we re-diffed everything! On the bright side of things, this small bug helped us refactor all the selected stuff and now it's fast even with this bug in!

Reply:
![](https://media.giphy.com/media/iXUbgnxzw7Xr10kMd7/giphy.gif)